k8s/examples/wordpress/wordpress/20-statefulset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: wordpress
+spec:
+  selector:
+    matchLabels:
+      app: wordpress
+      tier: frontend
+      storageos.com/replicas: "1"
+  serviceName: wordpress
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: wordpress
+        tier: frontend
+        storageos.com/replicas: "1"
+    spec:
+      serviceAccountName: wordpress
+      containers:
+      - name: wordpress
+        image: wordpress:5.2-apache
+        env:
+        - name: WORDPRESS_DB_HOST
+          value: wordpress-mysql
```

Comment:
I'm almost certain that this can't work without some sort of alteration. The MySQL service is Headless, therefore it is required to be referenced by pod-name.svc-name. In this case we are only using svc name.